### PR TITLE
CVE-2022-23184

### PR DESCRIPTION
**Pull request by OctoJames**
Updating CVE entry for SA2022-02

### CODE REVIEW

File: 2022/23xxx/CVE-2022-23184.json [hunk 1]
```diff
@@ -4,15 +4,64 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2022/23xxx/CVE-2022-23184.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/23xxx/CVE-2022-23184.json:4]: ("Expecting ',' delimiter: line 25 column 9 (char 690)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/23xxx/CVE-2022-23184.json (hunk 1 above)
Comment (OctoJames) [reply 2022/23xxx/CVE-2022-23184.json:4]: resolved missing delimiter

**Pull request merged**